examples/qvgviewer/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+list(APPEND TARGET_HEADERS
+    MainWindow.h)
+
+list(APPEND TARGET_SOURCES
+    MainWindow.cpp
+    main.cpp)
+
+list(APPEND TARGET_RESSOURCES
+    qvgviewer.qrc)
+
+list(APPEND TARGET_SVGS
+    svg/01.01.10.svg
+    svg/01.03.04q.svg
+    svg/01.08.05q.svg
+    svg/01.25.18.svg)
+
+list(APPEND TARGET_QVGS
+    qvg/01.01.10.qvg
+    qvg/01.03.04q.qvg
+    qvg/01.08.05q.qvg
+    qvg/01.25.18.qvg)
+
+set(TARGET_NAME qvgviewer)
+
+add_executable(${TARGET_NAME}
+    ${TARGET_SOURCES}
+    ${TARGET_HEADERS}
+    ${TARGET_RESSOURCES}
+    ${TARGET_SVGS})
+
+set_target_properties(${TARGET_NAME}
+    PROPERTIES
+        AUTOMOC ON
+        AUTORCC ON
+        FOLDER examples)
+
+target_link_libraries(${TARGET_NAME}
+    PRIVATE
+        qskinny
+        qsktestsupport)
+
+option(autoqvg "enable automatic qvg commpilation" OFF)
+
+if(autoqvg)
+    svg2qvg(${CMAKE_CURRENT_LIST_DIR}/svg/01.01.10.svg
+            ${CMAKE_CURRENT_LIST_DIR}/qvg/01.01.10.qvg)
+    svg2qvg(${CMAKE_CURRENT_LIST_DIR}/svg/01.03.04q.svg
+            ${CMAKE_CURRENT_LIST_DIR}/qvg/01.03.04q.qvg)
+    svg2qvg(${CMAKE_CURRENT_LIST_DIR}/svg/01.08.05q.svg
+            ${CMAKE_CURRENT_LIST_DIR}/qvg/01.08.05q.qvg)
+    svg2qvg(${CMAKE_CURRENT_LIST_DIR}/svg/01.25.18.svg
+            ${CMAKE_CURRENT_LIST_DIR}/qvg/01.25.18.qvg)
```

Comment:
@uwerat again generated files should never be in the source directory and should rather be placed in the build directory

Reply:
It is not mandatory to have a separate build directory with qmake ( and other build environments ). When working without an IDE it is actually pretty annoying to have different working directories for editing and building. So I made the qmake files to work with and without shadow builds.

However in the meantime I created my own type of "IDE" ( some shell scripts ) that allow me to build in a different directory without having to change my working directory.